taskfiles/k8s.yml

```diff
@@ -239,3 +239,16 @@ tasks:
     cmds:
       - 'echo "~~~ Creating operator v2 helm e2e tests :k8s:"'
       - ./hack/v2-helm-setup.sh
+
+  package-and-publish-operator-chart:
+    cmds:
+      - defer: 'helm registry logout registry-1.docker.io'
+      - helm registry login registry-1.docker.io -u {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}
+      - mkdir helm-operator-release
+      - defer 'rm -r helm-operator-release'
+      - yq -i '.name = "redpanda-operator-nightly"' charts/operator/Chart.yaml
+      - helm package charts/operator --version "{{.TAG_NAME}}-helm-chart" --destination helm-operator-release --dependency-update
```

Comment:
Ah right, I forgot this had to be renamed as well. Could you include an example command of how one might use the nightly build?

Reply:
I'm not sure if I should copy what is in official helm documentation how to use helm repository where chart's are stored?
 
 I can make a comment to reference official docker hub announcement https://www.docker.com/blog/announcing-docker-hub-oci-artifacts-support/

I could point to our artifact https://hub.docker.com/layers/redpandadata/redpanda-operator-nightly/v0.0.0-20250104git4a5a076-helm-chart/images/sha256-ffaea8752b6bd00a26589a168830a87d498106e470f11af0f08267bc13fbd078
where you can find all commands 
![Screenshot 2025-01-07 at 13 19 14](https://github.com/user-attachments/assets/81612623-1767-47a4-88f4-76ae263e3699)

What would you prefer?

Reply:
> where you can find all commands

Oh that's awesome! Yeah a link out to that page would be perfect.

I think it'd be good to include a note (or link to that forum post you had found) about why the tagging is suffixed with `-helm-chart`.